### PR TITLE
Add missing tf.tanh() tests for SparseTensor

### PR DESCRIPTION
**Pull request by siddharth-agrawal**
More information in comment [here](https://github.com/tensorflow/tensorflow/pull/2998#issuecomment-229743288). Tested locally.

### CODE REVIEW

File: tensorflow/python/kernel_tests/cwise_ops_test.py [hunk 1]
```diff
@@ -220,8 +222,10 @@ def testFloatBasic(self):
   def testFloatTanhEdge(self):
     x = np.arange(40, 40 + 6).reshape(6).astype(np.float32)
     self._compareBoth(x, np.tanh, tf.tanh)
+    self._compareBothSparse(x, np.tanh, tf.tanh)
     x = np.arange(-40, -40 + 6).reshape(6).astype(np.float32)
     self._compareBoth(x, np.tanh, tf.tanh)
+    self._compareBothSparse(x, np.tanh, tf.tanh, thresh=-40)
```
Comment (rmlarsen) [inline tensorflow/python/kernel_tests/cwise_ops_test.py]: With a threshold of -40 the sparsification will do nothing. Is that intentional?

Comment (siddharth-agrawal) [inline tensorflow/python/kernel_tests/cwise_ops_test.py]: Yes, I think the point of this test is to check numerical stability of the Tanh for large values of x, so I chose to not sparsify.

Comment (rmlarsen) [inline tensorflow/python/kernel_tests/cwise_ops_test.py]: @siddharth-agrawal I see. In that case I don't see any point in this change at all, since the dense and sparse versions call the same underlying tanh function, and there are already tests for the sparse version in the non-edge case.

Comment (siddharth-agrawal) [inline tensorflow/python/kernel_tests/cwise_ops_test.py]: @rmlarsen Yeah, that's true. Sorry for this PR.
